bot/constants.py

```diff
@@ -25,6 +25,11 @@ class Channels(NamedTuple):
     devlog = int(os.getenv("CHANNEL_DEVLOG", 789431367167377448))
 
 
+class Emojis(NamedTuple):
+    cucumber_emoji = "\U0001f952"
```

Comment:
```suggestion
    gurkan_emoji = "\U0001f952"
```
Gurkan sounds better ig.

Reply:
I disagree, since it is literally the cucumber emoji and not from the set of custom gurkan emojis.

Reply:
yeah, it can also create confusion on what the gurkan emoji even is, it could even be the gurkan crown in the future